core/src/main/java/com/linecorp/armeria/server/file/FileService.java

```diff
@@ -518,8 +519,9 @@ public boolean shouldCachePath(String path, @Nullable String query, Route route)
     enum ContentEncoding {
         // Order matters, we use the enum ordinal as the priority to pick an encoding in. Encodings should
         // be ordered by priority.
-        BROTLI(".br", "br", StreamDecoderFactory.brotli()),
-        GZIP(".gz", "gzip", StreamDecoderFactory.gzip());
+        BROTLI(".br", StreamDecoderFactory.brotli()),
+        GZIP(".gz", StreamDecoderFactory.gzip()),
+        SNAPPY(".sz", StreamDecoderFactory.snappy());
```

Comment:
Question) Is this actually intended for use? If it is, would it make sense to add it to `FileService.ContentEncoding#availableEncodings`?